MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+
+{% tabs %}
+{% highlight c# %}
+DataTable table = this.GetDataTable();
+this.sfDataGrid1.ItemsSource = table;
+{% endhighlight %}
+{% endtabs %}
+
+### Limitations
+
+* Custom sorting is not supported.
+* [SfDataGrid.View.Filter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_Filter) is not supported.
+* [SfDataGrid.View.LiveDataUpdateMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_LiveDataUpdateMode) is not supported.
+
+## View
+
+The `SfDataGrid` has the View property of type [ICollectionViewAdv](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html) interface that implements `ICollectionView` interface. View is responsible for maintaining and manipulating data and other advanced operations, like Sorting, Filtering, and etc.,
+
+When you bind collection to the `ItemsSource` property of the `SfDataGrid`, then View will be created and maintains the operations on Data such as Sorting, Filtering, Insert, Delete, and Modification.
+
+>Note: The `SfDataGrid` creates different types of view derived from `ICollectionViewAdv` interface based on the `ItemsSource` property.
+
+### LiveDataUpdateMode
+
+During data manipulation operations and property changes, the `SfDataGrid` supports updating the view by using the `LiveDataUpdateMode` property which exists in the [SfDataGrid.View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv) class.
+
+<table>
+<tr>
+<th>LiveDataUpdateMode</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>Default</td>
+<td>Data operations are not updated. </td>
+</tr>
+<tr>
+<td>AllowSummaryUpdate</td>
+<td>Summaries are updated during data manipulation change.</td>
+</tr>
+<tr>
+<td>AllowDataShaping</td>
+<td>Data operations like sorting, and filtering are updated during data manipulation change.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.View.LiveDataUpdateMode = LiveDataUpdateMode.Default;
+{% endhighlight %}
+{% endtabs %}
+
+### RecordPropertyChanged
+
+The [RecordPropertyChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_RecordPropertyChanged) event is invoked when the value of `DataModel` property is changed, if `DataModel` implements the `INotifyPropertyChanged` interface. The event receives with two arguments namely `sender` that handles the `DataModel` and the [PropertyChangedEventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.propertychangedeventargs?view=net-6.0) as argument. `PropertyChangedEventArgs` object contains the following property,

```

Comment:
The [RecordPropertyChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_RecordPropertyChanged) event is invoked when the value of the `DataModel` property is changed, if `DataModel` implements the `INotifyPropertyChanged` interface. The event receives with two arguments namely `sender` that handles the `DataModel` and the [PropertyChangedEventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.propertychangedeventargs?view=net-6.0) as argument. `PropertyChangedEventArgs` object contains the following property:

Reply:
Modified